src/lib/data/buyables/buyables.ts

```diff
@@ -1106,6 +1106,16 @@ const Buyables: Buyable[] = [
 			thieving: 50
 		}
 	},
+	{
+		name: 'Diving apparatus',
+		gpCost: 0,
```

Comment:
This may let people buy an infinite amount. I would set a cheap GP price, say 1k each.